configs/src/deployments/json/linea.json

```diff
@@ -185,5 +185,177 @@
+      "deploymentTx": "0xe80d5f25162e9ab20792c5d72a64fc38eb51f1815938aba5cfb422d24e02ded9"
+    },
+    "ManagementDAOMultisigProxy": {
+      "address": "0x45e24DcDc1ebc2a334dE20eEb114f00f96429AFa",
+      "blockNumber": 12678872,
+      "deploymentTx": "0xd8f789f33334efed9f0db20eee20e9b6caad978f831e60142698159ece87cffb"
+    },
+    "ENSRegistry": {
+      "address": "0xCe0B4124dea6105bfB85fB4461c4D39f360E9ef3",
+      "blockNumber": 12678710,
+      "deploymentTx": "0xda8dc8df2771f411c63d10da8fd1b24713237fbe190f8d064a0a4337a1cfb3cd"
+    },
+    "PublicResolver": {
+      "address": "0xE640Da5AD169630555A86D9b6b9C145B4961b1EB",
+      "blockNumber": 12678713,
+      "deploymentTx": "0x4d81e131793b31c2cf1e95f71880a3f3814e4f83751ff52d5d6a91eb48c001d1"
+    },
+    "DAOENSSubdomainRegistrarImplementation": {
+      "address": "0x5B3B36BdC9470963A2734D6a0d2F6a64C21C159f",
+      "blockNumber": 12678739,
+      "deploymentTx": "0x77610040a63b37ed4ca104f999c05cec586648c4fd66a42492441b3ffe3fb843"
+    },
+    "DAOENSSubdomainRegistrarProxy": {
+      "address": "0x96E54098317631641703404C06A5afAD89da7373",
+      "blockNumber": 12678743,
+      "deploymentTx": "0x1d99027d90cf08c5071f33fa3ead0bd7d1bd447670b0001908f0cd9519936bb6"
+    },
+    "PluginENSSubdomainRegistrarImplementation": {
+      "address": "0xE978942c691e43f65c1B7c7F8f1dc8cDF061B13f",
+      "blockNumber": 12678746,
+      "deploymentTx": "0x3608d0529e528c80b866fad5dcfb03955afe592d3bbef00dea618c22c9419e89"
+    },
+    "PluginENSSubdomainRegistrarProxy": {
+      "address": "0xA03C2182af8eC460D498108C92E8638a580b94d4",
+      "blockNumber": 12678748,
+      "deploymentTx": "0xc999ff10b8b258b7fc03e4df8c18ec3935ae02ba12da2704f8583895ff3b800e"
+    },
+    "DAORegistryImplementation": {
+      "address": "0x58cd7d773389F68250338C13A1BB5662CFBd0C9A",
+      "blockNumber": 17120762,
+      "deploymentTx": "0xe39c7e050de561be2a574aea162cd16dcadcff476c59e158b757cbe6ec4ac551"
+    },
+    "DAORegistryProxy": {
+      "address": "0xB2CCCB4d7ec56F992219BCF4abe2E7F02003443e",
+      "blockNumber": 12678757,
+      "deploymentTx": "0x76d31a2153f034d369eccd3ad6bf271e4d2e54290fedda387c5f7ce9711b28b8"
+    },
+    "PluginRepoRegistryImplementation": {
+      "address": "0xB815791c233807D39b7430127975244B36C19C8e",
+      "blockNumber": 17120765,
+      "deploymentTx": "0xdadab4055657cb41c05ec8dbd1aeb084a56bbe5b17b585fcf9e7b14dbc3fcd16"
+    },
+    "PluginRepoRegistryProxy": {
+      "address": "0xB2A2b32b9d885C85d5b229C0509341c37CaE7483",
+      "blockNumber": 12678764,
+      "deploymentTx": "0xf7ff02ca31149dc3be7e53ddd39e089b6f2248cc981006bef7b117b24a344abc"
+    },
+    "PluginRepoFactory": {
+      "address": "0x7870837ffe670E62d4e601393D454f1b8649F7f9",
+      "blockNumber": 17120759,
+      "deploymentTx": "0x52720be75ec7128badbf60dc2b661b50259630465875aa50b1b1207f1e567aaa"
+    },
+    "PluginRepoBase": {
+      "address": "0x3b4D91ed7F1A8a2648Cc72E8dE31f0d35A4f9A65",
```

Comment:
All existing plugins point to [0x31ee445ef8404ff04d8f7bc3d94b2e4b61ee1186](https://lineascan.build/address/0x31ee445ef8404ff04d8f7bc3d94b2e4b61ee1186#code)

- Remove this or
- Set the current value

Reply:
https://github.com/aragon/osx-commons/pull/142/files#r2012850630